### PR TITLE
Added Data Augmentation.ipynb

### PR DESCRIPTION
**Pull request by So-ham**

### CODE REVIEW

File: notebooks/chapter24/Data Augmentation.ipynb [hunk 1]
```diff
@@ -0,0 +1,483 @@
...
...sfvD+A8sNGunHQDQW9CKSQklLlpEhjmpzwVQzbkUftZebVuPiR5pZwacb3FICGFBmcMEhUCaMMsoQKaEnhqzRIupmNbaKl1bqAMdqXjLUx+N4Yyr01xvjwPgeTW78dufvaLLBYzLFYnlBtYUm1kyCMD2wIObsadk49JifHwPERYF0DrQGtAWOAv/31AHl+gMmVz7G/L7C/f93PtK7gnMOzrx4iy4DBgKoH7u7u4sqVKxgUA8ikxGI2w9HRFEdHR7F+VXg43fvgjq+6eISjoxNMp1PUS+2FK7C3twclJGYnDZ48+QIPHz7Dk8+pDV2WwJXyJnRjcXw8w3JhMKhu4Wc/pInE92+8hbIYYzS8iqocI0urGKGklEI+uOE9lwbzE0cx6FJAqQKpynw0VQqBxIcfZpSQ4EXqUJ8SLIv22+FChdvOCXWQ9ARPDIR1yNIS1iVQJvWdQyWyZgBtalhrIOpVW49KrTQwNlgwEwPjrV8OAg7WSw2p+oVzBlIBgMNyOceynsOYGsY2vhAajRWHuF4HA1drGNPECv30AKBgdCFcLF+zWMxweOBwePAMxj5DXQN1DaiMRE7JF0fI8yMUBZBlwJdf0rqmoWuT58BoBOzskPDEn/6FkxPg+XPg8NDvT4W6v4Bzz6lnOKGCaGUJXN0pfaCIxDs3fovre/u4du0GRsMdDAZjDKsrKMsRpicLVOUEeVahri1OjucwtUM1HGNnZ4JpPYOE9WO1ABCCISiCKVEFKNdVecF2vU4XCqPF1EG2uN8Wbyy2K+RJOlCKVVmOyNW1pbemNbSpo6urGkSLG8qFkivcxM8pIojautauLDAAQDeYzWa0L6UgpYhiDdkyFAzi/LQfFKhR1zXySRNd1/BQcD7AIYxf0DSJFIJYN9ThFiKtDr7+4lRdp9ArfO2DJMYWh86l4E4vpw6Ju4tKSoxv5MjeylAUlIOa56WfeoUqZBR5heFwJ86yUBS0XjR36ViS6oQZ7WA0cPJcIEuvo54pmEWCwWCMnZs0m95y1uDgqylkQSVcEiVj+KH05UYF6BpG3Ib4YLa2r403IFwfLBDcZUeFt6pyHNuoZEmbKFrnHDKXRPGEz+nVi9VbwZCuFkQcBdfM4lQjIYAhdPpIhSgYY0ISQGiHA4cnVMuYMmR8ZoyuyY12GsvlEkrR9tYan4jgkGbU+3v39q+j5W8TCsAlSU
```
Comment (antmarakis) [inline notebooks/chapter24/Data Augmentation.ipynb]: I am afraid this passage is hard to follow as well. Your text needs to follow a logical path from sentence to sentence, here it reads more like you stitched different pieces of information together. For example, The 'removing noise' line should come after the 'removing high frequency' line, since the first and third sentences are closely related in terms of meaning. In general, I would like to see this rewritten, if possible.

**Comment by So-ham** [comment]
> This is interesting and useful work, but I am afraid at the moment it is a bit bare-bones. If you have the time, I would like to see this fleshed out a bit more. For example, you do not explain the techniques used here at all, you simply show as the code. I think the notebook would benefit massively from some description.



> This is interesting and useful work, but I am afraid at the moment it is a bit bare-bones. If you have the time, I would like to see this fleshed out a bit more. For example, you do not explain the techniques used here at all, you simply show as the code. I think the notebook would benefit massively from some description.

Can you be a bit more specific about "explaining the techniques" please! Should I describe (logic) the code used in all cases or describe the sub-parts of each augmentation type like elaborating each type of blurring , noise used in the notebook or both? It would help in making the appropriate changes. Thanks for the review!

**Comment by antmarakis** [comment]
I would say you need to give the reader the background information on the technique (for example, what is 'translation'), the code doesn't need to be explained too much. It's mostly the theory that is lacking.

**Comment by So-ham** [comment]
Sure thanks!

**Comment by antmarakis** [comment]
Also, I tried to close-reopen the PR (so that the tests would be rerun), but for some reason Github is glitching and I cannot reopen it.

Can you try opening it?

**Comment by antmarakis** [comment]
Oh, I think the problem is that you deleted your fork. I am afraid you have to create a new one, in this case.

**Comment by So-ham** [comment]
Yes I did delete the fork by mistake but creating a new pr wouldn't be an issue as i have the notebook backed-up locally.
Will do it after the suggested changes have been made!
